apps/emqx/src/emqx_schema.erl

```diff
@@ -1929,6 +1929,15 @@ common_ssl_opts_schema(Defaults) ->
                     default => Df("secure_renegotiate", true),
                     desc => ?DESC(common_ssl_opts_schema_secure_renegotiate)
                 }
+            )},
+
+        {"hibernate_after",
+            sc(
+                duration(),
+                #{
+                    default => Df("hibernate_after", "5s"),
```

Comment:
here follows emqx 4.x that has default `5s` and personally I think it will be much easier to make it default to be disabled `undefined`

Reply:
Personally, I'd recommend using `infinity` instead of `undefined` for this.
Although their effect is the same.
But our other configurations are using `infinity` 
such as 
```
 {"idle_timeout",
            sc(
                hoconsc:union([infinity, duration()])
                ...
```
https://www.erlang.org/doc/man/erlang.html#type-timeout
```
hibernate_after() = timeout()
timeout() = infinity | integer() >= 0
```

By the way, what happens if we set it to 0, is 0 the same as infinity? 
Maybe we should also mention this in the document.